examples/stable_diffusion_v2/docs/en/controlnet.md

```diff
@@ -276,7 +276,7 @@ Here are some inference results after training Fill50k dataset (lr=5e-4, bs=2, e
 ![controlnet_train_validate](https://github.com/congw729/mindone/assets/115451386/62ad40a3-0510-4606-84cb-780c72989b36)
 
 Comparing the results with ground truth:
-![controlnet_train_gt](https://github.com/congw729/mindone/assets/115451386/644e436c-4ac9-4b23-81ff-6c0f0dbef9f8)
+![controlnet_train_gt](https://github.com/congw729/mindone/assets/115451386/b0e5740c-2488-4924-842f-72afcfc0abea)
 
 
```

Comment:
Some advice related to readme file:
1. It seems some configuration files are no longer needed, for example. [examples/stable_diffusion_v2/configs/v1_inference_contorlnet.yaml](https://github.com/mindspore-lab/mindone/blob/96758063c18981df1263d0e28acf08bfa6de3306/examples/stable_diffusion_v2/configs/v1_inference_contorlnet.yaml#L4). You may consider removing this file to remove redundancy. 
2. [examples/stable_diffusion_v2/configs/train/sd15_controlnet.yaml](https://github.com/mindspore-lab/mindone/blob/916f3669766b4939f08b5914e865da48bb7ed9a0/examples/stable_diffusion_v2/configs/train/sd15_controlnet.yaml) should be renamed to  `sd15_controlnet_canny.yaml`, so are the readme file's mentions.
3. Please mention `sd15_controlnet_openpose.yaml` in the readme file to the users when they want to finetune with 
pose control.

Reply:
1. done